engines/python/setup/djl_python/deepspeed.py

```diff
@@ -128,8 +128,7 @@ def __init__(self):
         self.ds_config = None
         self.task = None
         self.logger = logging.getLogger()
-        self.model_dir = None
-        self.model_id = None
+        self.pretrained_model_name_or_path = None
```

Comment:
I feel `model_id_or_path` is better, we exposed `model_id` key in the serving.properties to match huggingface, using model_name here is not consistent.

Reply:
Good point, will update.